src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -213,12 +213,25 @@ class SystemCves extends Component {
     };
 
     showStatusModal = cves => {
+
+        let hasDifferentStatus = false;
```

Comment:
Do you really have to initialize `hasDifferentStatus=false`.  `some()` returns false if the callback of the function returns a `falsy` value